src/util/getToken.js

```diff
@@ -1,8 +1,16 @@
-export default function getToken(withBearer = false) {
+/**
+ * Returns the JWT token this is stored in localStorage under key '_tk'
+ * @param {boolean} withBearer should be set to TRUE if the 'Bearer' part of the token string is required
+ * @param {string} key the localStorage key to get stored token from
+ */
+export default function getTokenFromStorage(withBearer = false, key = '_tk') {
   if (typeof window === 'undefined') return null;
 
-  const token = window.localStorage.getItem('_tk');
+  const token = window.localStorage.getItem(key);
   if (token === null) return null;
 
-  return withBearer ? token : token.substr(7);
+  if (withBearer) {
+    return token;
+  }
+  return token.substr(7);
```

Comment:
this was changed in allianz-

```js
export default function getToken(withBearer = false) {
  if (typeof window === 'undefined') return null;

  const token = window.localStorage.getItem('_tk');
  if (token === null) return null;

  return withBearer ? token : token.substr(7);
}
```

Reply:
the above won't error out if `token` is some non string non null value (it happened)

Reply:
updated token check